### PR TITLE
Dev web

### PR DESCRIPTION
**Pull request by giacomo270197**

### CODE REVIEW

File: tubuddy_web/api/forms.py [hunk 1]
```diff
@@ -5,7 +5,7 @@ class PatientForm(forms.Form):
     name = forms.CharField(widget=forms.TextInput(attrs={'class': 'form-control'}), label='Name', max_length=100)
     username = forms.CharField(widget=forms.TextInput(attrs={'class': 'form-control'}), label='Username', max_length=100)
     password = forms.CharField(widget=forms.PasswordInput(attrs={'class': 'form-control'}), label='Password', max_length=100)
-    api_token = forms.CharField(widget=forms.TextInput(attrs={'class': 'form-control'}), label='Api_token', max_length=300)
+    # api_token = forms.CharField(widget=forms.TextInput(attrs={'class': 'form-control'}), label='Api_token', max_length=300)
```
Comment (StenSipma) [inline tubuddy_web/api/forms.py:8]: So this is no problem??

File: tubuddy_web/api/forms.py [hunk 2]
```diff
@@ -20,7 +20,7 @@ class RegisterForm(forms.Form):
     name = forms.CharField(widget=forms.TextInput(attrs={'class': 'form-control'}), label='Name', max_length=100)
     password = forms.CharField(widget=forms.PasswordInput(attrs={'class': 'form-control'}), label='Password',
                                max_length=100)
-    api_token = forms.CharField(widget=forms.TextInput(attrs={'class': 'form-control'}), label='Api_token', max_length=100)
+    # api_token = forms.CharField(widget=forms.TextInput(attrs={'class': 'form-control'}), label='Api_token', max_length=100)
```
Comment (StenSipma) [inline tubuddy_web/api/forms.py:23]: Same for this?

Comment (giacomo270197) [reply tubuddy_web/api/forms.py:23]: not that I know of

Comment (StenSipma) [reply tubuddy_web/api/forms.py:23]: Oh nevermind, I thought this was in our code.

'not our problem' ;)

**Review by StenSipma — APPROVED** [review]

**Pull request merged**